README.md

```diff
@@ -2,6 +2,11 @@
 
 "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA)
 
+### project spashoots
+![math1](https://user-images.githubusercontent.com/60146030/177217028-758cf52e-39c3-4b01-aa9f-afebc8916816.JPG)
+![math2](https://user-images.githubusercontent.com/60146030/177217050-c427d549-ddba-4e7a-91fd-2f807a3ffb2b.JPG)
+![math3](https://user-images.githubusercontent.com/60146030/177217076-7cfa1480-ea2b-48a9-83a8-ba6f4742a6de.JPG)
+
```

Comment:
- [ ]  optional 

I strongly suggest that you deploy your finished project to netlify for visitor to view to demo of your project

Reply:
thanks, I think this optional suggestion is the upcoming lessons